content/rc/how-to/flush-db.md

```diff
@@ -1,22 +1,23 @@
 ---
-Title: Flushing Databases on Redis Cloud Essentials
+Title: Flushing Databases
 description:
 weight: $weight
 alwaysopen: false
 categories: ["RC"]
 ---
 There are times where you want to delete all database data.
 
-{{% warning title="Data Loss Warning" %}} The flush command deletes ALL of the data in the database. This
-includes all data in memory and persisted to disk. We recommend that you
-[backup your database]({{< relref "/rc/administration/configuration/backups.md" >}}) first.{{% /warning %}}
+{{% warning title="Data Loss Warning" %}}
+The flush command deletes ALL of the data in the database.
```

Comment:
We've already put this important information in a warning, the highest-level admonition. Personally, I don't like using all caps as visual emphasis in documentation.